.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,3 +15,4 @@ if you have questions about contributing.
 - [ ] Resolves #? <!-- issue must be created for each patch -->
 - [ ] tests added and passing
 - [ ] module layout described at `docs/development/architecture.rst` is up-to-date <!-- if you have added, renamed or removed files or directories please update the documentation accordingly -->
+- [ ] added PR title and github handle to release notes for next major release
```

Comment:
Why is major release only? What about minor and patch releases?

Reply:
I think it's because our minor release notes are usually a subset of the major release notes (i.e. we include all of the changes that go into a minor release in a major release, so it shouldn't be too difficult to generate minor release notes from this doc)

Reply:
Although maybe we should have a minor release notes template instead, and just compile all of them into one for the major release? I personally prefer the solution @devin-petersohn is implementing here, but thought I'd throw this idea out there

Reply:
I'm not sure it's always obvious if a PR is going to be backported. At first, I think it's easier to just have all the release notes in one place. I think we can revisit the minor release notes after this is working well.